scripts/azure-pipelines/linux-build_and_test.sh

```diff
@@ -16,21 +18,40 @@ function or_die () {
     fi
 }
 
-or_die mkdir azure-${TEST_TARGET}-build
-cd azure-${TEST_TARGET}-build
-if [[ "$DO_BUILD" == "yes" ]] ; then
-    or_die cmake -DCMAKE_CXX_COMPILER="${COMPILER}" ${CMAKE_EXTRA_FLAGS} ../src
-    if [[ ${CMAKE_EXTRA_FLAGS} == *COVERAGE* ]] ; then
-      or_die make -j 3
-    else
-      or_die make -j 3 VERBOSE=1
-    fi
-    if [[ "${DO_TEST}" == "yes" ]] ; then
-      or_die ctest -T test --output-on-failure -V
-    fi
-    if [[ "${DO_MEMCHECK}" == "yes" ]] ; then
-      or_die ctest -T memcheck
-    fi
-fi
+or_die cd axom
+git submodule init 
+git submodule update 
+
+echo HOST_CONFIG
+echo $HOST_CONFIG
+
+./config-build.py -hc /home/axom/axom/host-configs/docker/${HOST_CONFIG}.cmake -DENABLE_GTEST_DEATH_TESTS=ON
+
+or_die cd build-$HOST_CONFIG-debug
+or_die make -j 3
```

Comment:
Do you know how many threads we can use on azure pipelines?

It looks like we're building with 3 threads and runing tests with 8.

Reply:
I think we can run up to 10 parallel jobs.
I don't know how many threads of operation there are in a job.
https://docs.microsoft.com/en-us/azure/devops/pipelines/licensing/concurrent-jobs?view=azure-devops

Reply:
Thanks -- in that case, should we increase this to ``j10``?